modin/pandas/test/test_io.py

```diff
@@ -984,6 +1043,10 @@ def test_dataframe_to_csv(self, request):
             modin_obj=modin_df, pandas_obj=pandas_df, fn="to_csv", extension="csv"
         )
 
+    @pytest.mark.skipif(
+        Backend.get() == "Omnisci",
+        reason="to_csv is not implemented with OmniSci backend yet",
```

Comment:
maybe submit a tracker and reference it here?

Reply:
Fixed.